python/src/LinearModelStepwiseAlgorithm_doc.i.in

```diff
@@ -59,10 +59,10 @@ Examples
 >>> basis = ot.Basis(functions)
 >>> '''Stepwise regression'''
 >>> minimalIndices = [0]
->>> isForward = True
->>> penalty = 2.0 #Akaike information criterion 
->>> maximumIterationNumber = 10
->>> algo_forward = ot.LinearModelStepwiseAlgorithm(input_sample, basis, output_sample, minimalIndices, isForward, penalty, maximumIterationNumber)
+>>> direction = 1 #Forward selection
```

Comment:
it's better to use the enum name (ot.LinearModelStepwiseAlgorithm.FORWARD) rather than just its value

Reply:
I was indeed wondering how to do that properly. Thanks for the tip!